src/us/kbase/typedobj/core/LocalTypeProvider.java

```diff
@@ -1,42 +1,32 @@
 package us.kbase.typedobj.core;
 
+import static java.util.Objects.requireNonNull;
+
 import us.kbase.typedobj.db.TypeDefinitionDB;
 import us.kbase.typedobj.exceptions.NoSuchModuleException;
 import us.kbase.typedobj.exceptions.NoSuchTypeException;
 import us.kbase.typedobj.exceptions.TypeStorageException;
 
 /** A type provider for the typed object validator that takes a direct instance
  * of a type database.
- * @author gaprice@lbl.gov
```

Comment:
they'll still be able to find you through the git blame

Reply:
It's more about futilely trying to avoid email harvesting bots